Source/EasyNetQ.Tests/EasyNetQ.Tests.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>EasyNetQ.Tests</Description>
     <LangVersion>latest</LangVersion>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>netcoreapp5.0</TargetFramework>
```

Comment:
Maybe `net5` instead of `netcoreapp5.0` ? https://github.com/dotnet/sdk/issues/13295

Reply:
that's for sure 😄 didn't saw it